Mathlib/Data/ZMod/Units.lean

```diff
@@ -50,7 +50,7 @@ theorem unitsMap_surjective [hm : NeZero m] (h : n ∣ m) :
     have : NeZero n := ⟨fun hn ↦ hm.out (eq_zero_of_zero_dvd (hn ▸ h))⟩
     simp [unitsMap_def, - castHom_apply]
   intro x hx
-  let ps := m.primeFactors.filter (fun p ↦ ¬p ∣ x)
+  let ps : Finset ℕ := {p ∈ m.primeFactors | ¬p ∣ x}
```

Comment:
Do you need the type ascription here? Just wondering

Reply:
No, I just assumed that it's less fragile this way.